From 911966c4a6510a8f17acd23bc8e2d6f3f56a95db Mon Sep 17 00:00:00 2001
From: Jerry Cao <jerry.cao@amlogic.com>
Date: Tue, 11 Mar 2014 14:20:31 -0700
Subject: [PATCH 3735/5965] Fix duplicate ZTE_PRODUCT_AC2726 entry.

ZTE_PRODUCT_AC2726 was added by Amlogic and defined as 0xfff5.
v3.10.33 has added ZTE_PRODUCT_AC2726 and defined as 0xfff1.

The 0xfff5 entry added by Amlogic is removed as a result of this merge.
---
 drivers/usb/serial/option.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/usb/serial/option.c b/drivers/usb/serial/option.c
index fd62ee6b54db..e44994394e74 100755
--- a/drivers/usb/serial/option.c
+++ b/drivers/usb/serial/option.c
@@ -338,7 +338,6 @@ static void option_instat_callback(struct urb *urb);
 #define ZTE_PRODUCT_CDMA_AC682			0xffdd
 #define ZTE_PRODUCT_CDMA_TECH			0xfffe
 #define ZTE_PRODUCT_AC8710			0xfff1
-#define ZTE_PRODUCT_AC2726			0xfff5
 #define ZTE_PRODUCT_AC8710T			0xffff
 #define ZTE_PRODUCT_MC2718			0xffe8
 #define ZTE_PRODUCT_AD3812			0xffeb
@@ -1819,7 +1818,6 @@ static const struct usb_device_id option_ids[] = {
 
 	{ USB_DEVICE_AND_INTERFACE_INFO(ZTE_VENDOR_ID, ZTE_PRODUCT_CDMA_TECH, 0xff, 0xff, 0xff) },
 	{ USB_DEVICE_AND_INTERFACE_INFO(ZTE_VENDOR_ID, ZTE_PRODUCT_AC8710, 0xff, 0xff, 0xff) },
-	{ USB_DEVICE_AND_INTERFACE_INFO(ZTE_VENDOR_ID, ZTE_PRODUCT_AC2726, 0xff, 0xff, 0xff) },
 	{ USB_DEVICE_AND_INTERFACE_INFO(ZTE_VENDOR_ID, ZTE_PRODUCT_AC8710T, 0xff, 0xff, 0xff) },
 	{ USB_DEVICE(ZTE_VENDOR_ID2, ZTE_PRODUCT_MF_330) },
 	{ USB_DEVICE_AND_INTERFACE_INFO(ZTE_VENDOR_ID, ZTE_PRODUCT_MC2718, 0xff, 0xff, 0xff),
-- 
2.19.0

